WooCommerce/Classes/POS/Presentation/ProductCardView.swift

```diff
@@ -40,6 +40,6 @@ struct ProductCardView: View {
 
 #if DEBUG
 #Preview {
-    ProductCardView(product: POSProductFactory.makeProduct(currencySettings: ServiceLocator.currencySettings))
+    ProductCardView(product: POSProductProvider.provideProductForPreview(currencySettings: ServiceLocator.currencySettings))
```

Comment:
nit: this isn't from this PR, would be nice not having ServiceLocator usage in the POS folder

(Same for `ProductRowView`)

```suggestion
    ProductCardView(product: POSProductProvider.provideProductForPreview(currencySettings: .init()))
```

Reply:
Agree! Changed here: https://github.com/woocommerce/woocommerce-ios/pull/12845/commits/94a06c5876a8316131b333238192ca80f4c3f139

On the same note, I don't think we should have this dependency at all here (in the product level, neither in the preview), but the currency is something that should be passed to all products from somewhere else, since different products shouldn't have different currency configuration. I'll add a note to update this on a different PR.